Obsidian/Entities/Entity.cs

```diff
@@ -342,4 +340,9 @@ public bool Equals([AllowNull] Entity other)
     public static bool operator !=(Entity a, Entity b) => !(a == b);
 
     public override int GetHashCode() => this.EntityId.GetHashCode();
+
+    //TODO IMPLEMENT THESE
```

Comment:
again, is this still TODO?

Reply:
Yeah, I'll fully write these out now actually